src/main/java/io/cdap/plugin/snowflake/sink/batch/CSVBuffer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.snowflake.sink.batch;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+import java.io.ByteArrayOutputStream;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * A buffer which the {@link CSVRecord} is written to before it gets sent to Snowflake.
+ */
+public class CSVBuffer implements Closeable {
+  private static final CSVFormat csvFormat = CSVFormat.DEFAULT
+    //.withQuoteMode(QuoteMode.ALL)
```

Comment:
why is this commented?

Reply:
this is unnecessary forgot to remove. It still quotes everything as expected. We had test cases where values had commas in them.